### ISSUE TITLE
winning bid in prebid

### ISSUE THREAD

**Issue by manishdhiman** [prebid/Prebid.js#1667]

**Comment by manishdhiman** [comment]
I have implemented header bidding though prebid, but i am not able to see winning bid into it, I can see all bids are sending on page, Please help me out how can i use sent all winning bids with syntax.

here is the tags are which is using on page:

pbjs.bidderSettings = {
                pubmatic: {
                    sendStandardTargeting: false,
                    suppressEmptyKeys:true,
                    alwaysUseBid:true,
                    adserverTargeting: [
                      {
                          key: "hb_size_pubmatic",
                          val: function(bidResponse) {
                              return bidResponse.size;
                          }
                      },{
                          key: "hb_adid_pubmatic",
                          val: function(bidResponse) {
						      return bidResponse.adId;
                          }
                        },{
                            key: "hb_pb_pubmatic",
                            val: function(bidResponse) {
                                return bidResponse.pbHg;
                            }
                        },{
                            key: "hp_bidder_pubmatic",
                            val: function(bidResponse) {
                                return 'pubmatic';
                            }
                        },{
                          key: "hb_bidder_pubmatic",
                          val: function(bidResponse) {
                              return "pubmatic";
                          }
                        }
                    ]
                 },appnexus: {
                     sendStandardTargeting: false,
                    suppressEmptyKeys:true,
                    alwaysUseBid:true,

                     adserverTargeting: [
                       {
                           key: "hb_size_appnexus",
                           val: function(bidResponse) {
							  return bidResponse.size;
                           }
                       }, {
                           key: "hb_adid_appnexus",
                           val: function(bidResponse) {
                               return bidResponse.adId;
                           }
                         },
                         {
                             key: "hb_pb_appnexus",
                             val: function(bidResponse) {
								return bidResponse.pbHg;
                             }
                         },{
                          key: "hb_bidder_appnexus",
                          val: function(bidResponse) {
                              return "appnexus";
                          }
                        }
                       
                     ]
                  },rubicon: {
					sendStandardTargeting: false,
                    suppressEmptyKeys:true,
                    alwaysUseBid:true,
                    adserverTargeting: [
                      {
                          key: "hb_size_rubicon",
                          val: function(bidResponse) {
							  return bidResponse.size;
                          }
                      },{
                          key: "hb_adid_rubicon",
                          val: function(bidResponse) {
								return bidResponse.adId;
                          }
                        },{
                            key: "hb_pb_rubicon",
                            val: function(bidResponse) {
								return bidResponse.pbHg;
                            }
                        },{
                          key: "hb_bidder_rubicon",
                          val: function(bidResponse) {
                              return "rubicon";
                          }
                        }
                    ]
                 },criteo: {
                     sendStandardTargeting: false,
                    suppressEmptyKeys:true,
                    alwaysUseBid:true,

                     adserverTargeting: [
                       {
                           key: "hb_size_criteo",
                           val: function(bidResponse) {
							   return bidResponse.size;
                           }
                       }, {
                           key: "hb_adid_criteo",
                           val: function(bidResponse) {
                               return bidResponse.adId;
                           }
                         },
                         {
                             key: "hb_pb_criteo",
                             val: function(bidResponse) {
								var cpm = bidResponse.cpm;
            					return (Math.floor(cpm)/65).toFixed(2);
                             }
                         },{
                          key: "hb_bidder_criteo",
                          val: function(bidResponse) {
                              return "criteo";
                          }
                        }
                       
                     ]
                  }
            };
            
        });

**Comment by manishdhiman** [comment]
![prebid](https://user-images.githubusercontent.com/8361199/31329206-9a500970-acf6-11e7-804f-71dd77403777.png)

Please find the attachment for the same.

**Comment by matthewlane** [comment]
The formatting on this issue is a bit hard to follow, do you have a link to the page we can take a look at if you are still seeing this problem?
